src/plugins/robots/drone/simulator/pointmass3d_drone_model.cpp

```diff
@@ -242,24 +250,43 @@ namespace argos
          (JR * m_cAngularVelocity.GetX() * fOmegaR / INERTIA.GetY()) +
          (m_cAngularVelocity.GetZ() * m_cAngularVelocity.GetX() * (INERTIA.GetZ() - INERTIA.GetX()) / INERTIA.GetY()),
          (fTorqueZ / INERTIA.GetZ()) +
-         (m_cAngularVelocity.GetY() * m_cAngularVelocity.GetX() * (INERTIA.GetX() - INERTIA.GetY()) / INERTIA.GetZ()),
+         (m_cAngularVelocity.GetY() * m_cAngularVelocity.GetX() * (INERTIA.GetX() - INERTIA.GetY()) / INERTIA.GetZ())
       };
-      /* update the velocity using trapezoid integration */
-      cAcceleration = 0.5 * (m_cAccelerationPrev + cAcceleration);
-      m_cVelocity += cAcceleration * GetPM3DEngine().GetPhysicsClockTick();
-      m_cAccelerationPrev = cAcceleration;
+      /* gyroscope sensor readings */
+      cAngularAcceleration.Set(
```

Comment:
Do `SENS_BIAS_STD_DEV` and `SENS_NOISE_MEAN` just represent "no noise added for the moment" or "uncalibrated" or?

Reply:
`SENS_NOISE_MEAN` is the mean value of the Gaussian distribution of the noise that affects the sensors. Since I use the same mean value (0) for both gyro and accel, I use a single variable to describe it. 

`SENS_BIAS_STD_DEV`  is the standard deviation of the Gaussian distribution of the bias. Again,   I use the same deviation value (1) for both gyro and accel here. 

However, if you like, I can use different variables like  `GYRO_NOISE_MEAN`, `ACCEL_NOISE_MEAN`, `GYRO_BIAS_STD_DEV`, and `ACCEL_BIAS_STD_DEV`.

Reply:
Yes, please split these up since they are unrelated (apart from having the same value at the moment).